lib/config/applyrule-targeted.cpp

```diff
@@ -63,21 +63,21 @@ const std::vector<ApplyRule::Ptr>& ApplyRule::GetTargetedServiceRules(const Type
 bool ApplyRule::AddTargetedRule(const ApplyRule::Ptr& rule, const String& targetType, ApplyRule::PerSourceType& rules)
 {
 	if (targetType == "Host") {
-		std::vector<const String *> hosts;
+		std::set<const String> hosts;
```

Comment:
Shouldn't be const, `std::set` already tries to prevent you from modifying contained elements.